financial-templates-lib/GasEstimator.js

```diff
@@ -0,0 +1,71 @@
+// This script gets the current recommended `fast` gas price from ethGasStation
+// to inform the Liquidator and dispute bot of a reasonable gas price to use.
+
+const fetch = require("node-fetch");
+const url = "https://ethgasstation.info/json/ethgasAPI.json";
+const { Logger } = require("./Logger");
+
+// If no updateThreshold is specified then default to updating every 60 seconds.
+class GasEstimator {
+  constructor(updateThreshold = 60) {
+    this.updateThreshold = updateThreshold;
+    this.lastUpdateTimestamp;
+    this.lastFastPriceGwei;
+
+    // If the script fails or the API resonce fails default to this value
+    this.defaultFastPriceGwei = 40;
```

Comment:
I think it might be worth passing in a `defaultFastPriceGwei` similar to how you pass in `updateThreshold` and then set a default.

In other words, is it worth changing constructor to `constructor(updateThreshold = 60, defaultFastPriceGwei = 40)`? I think it might be useful because different clients might want to set different defaults.

Reply:
ye, this is a good idea. I will change it.